utils/apiObjects.ts

```diff
@@ -0,0 +1,151 @@
+import { hexToDecimal, stringToFelt } from "./feltService";
+import { formatHexString } from "./stringService";
+
+const STARKNET = formatHexString(stringToFelt("starknet"));
+const DISCORD = formatHexString(stringToFelt("discord"));
+const TWITTER = formatHexString(stringToFelt("twitter"));
+const GITHUB = formatHexString(stringToFelt("github"));
+const PROOF_OF_PERSONHOOD = formatHexString(
+  stringToFelt("proof_of_personhood")
+);
```

Comment:
these const should not be here imo cause they can be use elsewhere (like in `twitter.tsx` ...)

Reply:
Ok I will create a const file